src/main/webapp/script.js

```diff
@@ -0,0 +1,98 @@
+var zoomedIn = false;
+var aqLayerOn = true;
+var routeVisible = false;
+
+function changeBackgroundImage(imagePath) {
+    document.body.style.backgroundImage = "url('" + imagePath + "')";
+}
+
+function getImagePath() {
+    path = "";
+    if (zoomedIn) {
+        path = "images/wagga_";
+    } else {
+        path = "images/nsw_";
+    }
+
+    if (aqLayerOn) {
+        path += "gradient";
+    } else {
+        path += "plain";
+    }
+
+    if (routeVisible) {
+        path += "_route.png";
+    } else {
+        path += ".png";
+    }
+
+    return path;
+}
+function toggleAQLayer() {
+    checkbox = document.getElementById("aq-layer-toggle");
+    
+    aqLayerOn = checkbox.checked;
+
+    changeBackgroundImage(getImagePath());
+}
+
+function hideRoutes() {
+    routeVisible = false;
+
+    document.getElementById("close-directions").style.visibility = "hidden";
+
+    document.getElementById("route-info").style.visibility = "hidden";
+
+    changeBackgroundImage(getImagePath());
+}
+function showRoutes() {
```

Comment:
see above for whitespace consistency :')

Reply:
Done